### ISSUE TITLE
should resolve the `input` (or even File object) instead of result from rollup?

### ISSUE THREAD

**Issue by tunnckoCore** [tunnckoCore/start-rollup#4]
/cc @deepsweet what you think? I'm not sure enough, because it not relay on the incoming file (it respects more `options.entry` not something that comes from `files` task for example).

But I think that it should finally resolve the filepath/file for the next task to be able to continue working in some file. For example, one may want to use rollup + buble, then pass the bundle to webpack task? Hm.

**Comment by deepsweet** [comment]
well, my initial thoughts about bundlers like Webpack were about just to run them without input/output. but if Rollup can return filepath(s) then it's technically possible to manually connect it with Webpack (for some reason which I don't get, but still), something like this:

```js
export const build = () => start(reporter())(
  files('build/'),
  clean(),
  rollup({
    entry: './main.js',
    dest: './build/bundle.js',
    sourceMap: 'inline',
    plugins: [buble()]
  }),
  ((input) => webpack({ ...webpackConfig, entry: input })())
)
```

currently `start-webpack` doesn't handle any input, but maybe it should. sometimes it's really hard to connect such a differents tools together without unified IO and API.

**Comment by tunnckoCore** [comment]
> such a differents tools together without unified IO and API.

Absolutely. Same case is with `start-browserify` :D I'm just crazy after last night around #bundlersHell.
